pkgs/applications/audio/audacity/default.nix

```diff
@@ -80,7 +80,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     (fetchpatch {
-      url = "https://github.com/audacity/audacity/pull/831/commits/007852e51fcbb5f1f359d112f28b8984a604dac6.patch";
+      url = "https://github.com/audacity/audacity/commit/7f8135e112a0e1e8e906abab9339680d1e491441.patch";
       sha256 = "0zp2iydd46analda9cfnbmzdkjphz5m7dynrdj5qdnmq6j3px9fw";
```

Comment:
Did you verify that the hash is still the same?

Reply:
> Did you verify that the hash is still the same?

Yep, I've built the package with this patch and audacity ran successfully.